crawl-ref/source/viewmap.cc

```diff
@@ -873,6 +874,32 @@ class UIMapView : public ui::Widget
         return ret;
     }
 
+#ifdef USE_TILE_LOCAL
+    // This function should not be called while a map command is processing
+    void start_far_viewing(coord_def viewed_location)
+    {
+        m_far_view = true;
+        update_far_viewing(viewed_location);
+    }
+
+    void stop_far_viewing()
+    {
+        m_far_view = false;
+
```

Comment:
There's this one out of place blank line here! Looks good tho. Noticed this issue myself and was thinking "I'm sure this used to work here"

Reply:
Thanks, I've removed the bland line